### ISSUE TITLE
This package depends on the debug module

### ISSUE THREAD

**Issue by amcolash** [gerhardberger/electron-online#1]
I was using this module in conjunction with [electron-builder](https://github.com/electron-userland/electron-builder) and was getting errors that it could not load the module `debug`.

I think this is because that was listed as a dev dependency instead of a dependency. Although it is not necessary for the core functionality, it should either be conditionally loaded or required.

I don't have time at the moment to make a pr on this but thought it would be good to document my experiences.

As a workaround I just installed `debug` as a dependency for my project.

**Comment by mayankvadia** [comment]
@amcolash I also getting same issue, is it way to install `debug` as dependency

**Comment by amcolash** [comment]
Yes just `npm install debug` I believe. It's been a while since I did this, so going off of memory
